frontend/components/NotificationComponent/index.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import {View} from 'react-native';
+import {Avatar, Button, Divider, Text} from 'react-native-paper';
+import styles from './styles';
+
+/*
+ * Props:
+ * - text: the notification text
+ * - positiveAction: the action to execute when the user makes a positive choice
+ * - negativeAction: the action to execute when the user makes a negative choice
+ */
+function NotificationComponent(props) {
```

Comment:
You might want to import props as `{prop1, prop2, prop3}` instead of just `prop` just to be consistent with the rest of the frontend components. The other option would be for everyone else to change how they import their props. You can take a look at other components to see how they are doing it